assets/scss/modules/_side-navigation.scss

```diff
@@ -75,6 +76,13 @@ Styleguide Side Navigation
   }
 }
 
+.side-nav__link--mobile-tall {
```

Comment:
I'm not a fan of modifiers on children. I have always applied a modifier to the base class, which gives the component a modifier rather than different modifiers within the component. I'd also say is `mobile` needed here? if it became tall in `medium` or `large` breakpoints you would have to change the name. Maybe `.side-nav--tall` works better?

So in the sass:
```
.side-nav--tall {
    .side-nav__link {
        ...
    }
}
```